modules/apps/object/object-service/src/main/java/com/liferay/object/internal/field/business/type/DateTimeObjectFieldBusinessType.java

```diff
@@ -154,8 +155,11 @@ private LocalDateTime _getLocalDateTime(
 		ZonedDateTime zonedDateTime = ZonedDateTime.of(
 			localDateTime, ZoneId.of(sourceTimeZoneId));
 
-		return LocalDateTime.ofInstant(
+		LocalDateTime localDateTimeOfInstant = LocalDateTime.ofInstant(
```

Comment:
Hi @yuriymr, since this is an issue specific to the `SalesforceObjectEntryManagerImplTest` class, it's safer to fix this inside that class. The `DateTimeObjectFieldBusinessType` class is widely used, so there's a high risk of impacting something if we make changes here.
I think you can move the fix logic to [SalesforceObjectEntryManagerImplTest#_addObjectEntry method](https://github.com/liferay/liferay-portal/blob/b6aba40fa9037d15dea3b46d588809a1f9af9275/modules/apps/object/object-storage-salesforce-test/src/testIntegration/java/com/liferay/object/storage/salesforce/internal/rest/manager/v1_0/test/SalesforceObjectEntryManagerImplTest.java#L616-L619).